### PR TITLE
feat: add Front Matter docs, fix minor style buggs

### PR DESCRIPTION
**Pull request by RavenInd**
- add front matter docs
- correct cookie styles (no theming)
- provide short text for banner
- correct MDLinks color
- correct code snippets color

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -104,6 +104,19 @@ Parameters is passed in curly braces.
...
 `options-list` - required parameter. Comma separated strings, which is provided in tabs' `value` parameter. `value` - required parameter. It should be provided as an option in `options-list` in `tabsgroup`.
 
+### Front Matter
+
+The front matter must be the first thing in the markdown file and must take the form of valid YAML set between triple-dashed lines. Here is an example:
+
+```
+---
+title: Overview
+description: Hardhat | Ethereum development environment for professionals
```
Comment (usulpro) [inline docs/README.md:114]: These are not all possible options. please check what we have in md files

Comment (RavenInd) [reply docs/README.md:114]: done

File: docs/src/pages/[...docPath].tsx [hunk 2]
```diff
@@ -74,8 +74,8 @@ export const getStaticProps: GetStaticProps = async (props) => {
         seoDescription,
       },
       layout,
-      next,
-      prev,
+      next: data.next ?? next,
```
Comment (usulpro) [inline docs/src/pages/[...docPath].tsx]: do we allow only `next: false` in the frontmatter? what if user enters `next: true` - would this work in this case?

Comment (usulpro) [reply docs/src/pages/[...docPath].tsx]: @RavenInd

**Review by usulpro — APPROVED** [review]
LGTM with this comment https://github.com/focusreactive/hardhat/pull/58#pullrequestreview-983017741

File: docs/src/pages/[...docPath].tsx (hunk 2 above)
Comment (RavenInd) [reply docs/src/pages/[...docPath].tsx]: corrected

**Pull request merged**